### PR TITLE
In 150615102 view only mode for screenings

### PR DESCRIPTION
**Pull request by saraOSI**

### CODE REVIEW

File: spec/javascripts/components/screenings/ScreeningPageSpec.jsx [hunk 1]
```diff
@@ -32,6 +32,64 @@ describe('ScreeningPage', () => {
     spyOn(IntakeConfig, 'sdmPath').and.returnValue(sdmPath)
   })
 
+  describe('renderMode', () => {
+    it('uses the mode from params, if present', () => {
```
Comment (thomasramirez) [inline spec/javascripts/components/screenings/ScreeningPageSpec.jsx]: Having a mode in props that can be overridden by mode in props.params feels rather awkward.

Comment (saraOSI) [reply spec/javascripts/components/screenings/ScreeningPageSpec.jsx]: I'm not sure exactly why that is the case, but it was existing functionality. Maybe something to do with react router? I can dig into it a bit more.

Comment (saraOSI) [reply spec/javascripts/components/screenings/ScreeningPageSpec.jsx]: `props.params.mode` definitely comes from the react router. Which made my wonder why we would need the props directly, so I commented out the code, and TONS of tests failed. Maybe Nate has some historical context?

Comment (thomasramirez) [reply spec/javascripts/components/screenings/ScreeningPageSpec.jsx]: Given that this is existing functionality, I won't worry about it from a PR perspective but will be interested in an explanation at some point if anyone (probably Nate) has one.

File: spec/javascripts/components/screenings/participants/ParticipantCardHeaderSpec.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    onDelete = jasmine.createSpy('onDelete')
+    onEdit = jasmine.createSpy('onEdit')
+  })
+
+  function renderComponent({title = 'J Doe', showEdit = true}) {
+    const props = {title, showEdit, onEdit, onDelete}
+    return shallow(<ParticipantCardHeader {...props} />)
+  }
+
+  it('displays the name passed in the props', () => {
```
Comment (thomasramirez) [inline spec/javascripts/components/screenings/participants/ParticipantCardHeaderSpec.jsx]: Slightly boring but more precise to say it displays the title passed in the props

File: spec/javascripts/components/screenings/participants/ParticipantCardHeaderSpec.jsx [hunk 3]
```diff
@@ -21,6 +21,21 @@ describe('ParticipantCardHeader', () => {
     expect(component.text()).toContain('Alex')
   })
 
+  describe('participant flag', () => {
```
Comment (thomasramirez) [inline spec/javascripts/components/screenings/participants/ParticipantCardHeaderSpec.jsx]: better to describe as information flag as the component is `informationFlag`

File: app/javascript/screenings/ScreeningPage.jsx [hunk 4]
```diff
@@ -210,6 +224,8 @@ export class ScreeningPage extends React.Component {
     }
     const {screening, loaded} = this.state
     const mergedScreening = this.mergeScreeningWithEdits(this.state.screeningEdits)
+    const editable = this.editable()
```
Comment (NateOSI) [inline app/javascript/screenings/ScreeningPage.jsx]: This should be a prop in map state to props

**Pull request merged**